tests/Annotations/OpenApiRouteLoaderAnnotationsTest.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tobion\OpenApiSymfonyRouting\Tests\Annotations;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Exception\DirectoryNotFoundException;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+use Tobion\OpenApiSymfonyRouting\OpenApiRouteLoader;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\Basic\Controller as BasicController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\FormatSuffix\Controller as FormatSuffixController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\OperationId\Controller as OperationIdController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\PathParameterPattern\Controller as PathParameterPatternController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\Priority\Controller as PriorityController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\SeveralClasses\BarController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\SeveralClasses\FooController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\SeveralClasses\SubNamespace\SubController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\SeveralHttpMethods\Controller as SeveralHttpMethodsController;
+use Tobion\OpenApiSymfonyRouting\Tests\Annotations\Fixtures\SeveralRoutesOnOneAction\Controller as SeveralRoutesOnOneActionController;
+
+class OpenApiRouteLoaderAnnotationsTest extends TestCase
+{
+    private const FIXTURES_ROUTE_NAME_PREFIX = 'tobion_openapisymfonyrouting_tests_fixtures_annotations_';
+
+    public function testBasic(): void
```

Comment:
Instead of re-implementing the tests for attributes and annotations, I'd suggest to keep the current tests of OpenApiRouteLoaderTest but make it abstract. Then extend this base test class from OpenApiRouteLoaderAnnotationsTest and OpenApiRouteLoaderAttributesTest. Both tests cases should be exactly the same. Only the fixtures base path changes.

Reply:
It was some problems with swagger ^4.0 

Swagger by default generates hash for operationId by default 
see https://github.com/zircote/swagger-php/blob/master/src/Processors/OperationId.php#L23

Format for ^4.0 does not assert with the previous version, that's why I removed operationId generation checks. 

I think that abstract tests it bad idea, tests workflow will be changed for some environments and it will open some problems with changing the tests workflow if you have abstract tests for both environments.

Reply:
The operatinId processor was remove before and should also be removed in the attribute version. Analysis::processors does not exist anymore in 4.x it seems. But we can call Generator::getProcessors and Generator::setProcessors to remove the OpererationId processor: https://github.com/zircote/swagger-php/blob/3e41d604d6d1496043be1de9f688db410177f384/src/Generator.php#L183-L217
This should also work in 3.x to make it the same for attributes and annoatations.